src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +23,8 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    <ArchitectureSpecificToolFile Include="$(BinDir)libclrjit.so"/>
```

Comment:
In this change, this will be used for both runtime and crossgen. Once the JIT package comes along, we will get it from there for both the scenarios.